heartbeat/heartbeat.full.yml

```diff
@@ -346,20 +346,12 @@ output.elasticsearch:
   # so the final name is heartbeat-%{[beat.version]}.
   #template.name: "heartbeat"
 
-  # Path to template file
-  #template.path: "${path.config}/heartbeat.template.json"
+  # Path to fields.yml file to generate the template
+  #template.path: "${path.config}/fields.yml"
```

Comment:
Agree on rethinking the config options. I added it to https://github.com/elastic/beats/issues/3654